File: Python/tests/test_hgf_ibrb.py
import pytest

from hbayesdm.models import hgf_ibrb


def test_hgf_ibrb():
    _ = hgf_ibrb(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_hgf_ibrb.R
context("Test hgf_ibrb")
library(hBayesDM)

test_that("Test hgf_ibrb", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(hgf_ibrb(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
